src/main/java/com/example/hadconsentservice/service/PatientService.java

```diff
@@ -68,4 +69,18 @@ public ResponseEntity<Response> updateConsentStatus(ConsentRequest consentReques
 
     }
 
+    public String get_patientID_from_guardianID(String guardianID){
+        String patient_id;
+        Optional<Guardian> guardian = guardianService.findPatientById(Long.parseLong(guardianID));
+        try{
+            patient_id = String.valueOf(guardian.get().getAssignedPatientID());
+            return patient_id;
+        }
+        catch(Exception e){
+            return "";
+        }
```

Comment:
return e.getMessage()

Reply:
this is required for the flow of the program